### ISSUE TITLE
Project lint fails due to expired TODO comment in NameConfiguration.swift

### ISSUE THREAD

**Issue by kaseken** [realm/SwiftLint#6076]
### New Issue Checklist

- [x] I've Updated SwiftLint to the latest version.
- [x] I've searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues).

### Bug Description

The deadline of the following TODO comment has been reached and causes lint error.
https://github.com/realm/SwiftLint/blob/ad23d08cdaa7d656a6e96d6f6cc7b0d165f0a8f2/Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/NameConfiguration.swift#L77

Cloning this project and running `swiftlint lint .` will reproduce this issue.

```sh
/Users/username/Projects/SwiftLint/Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/NameConfiguration.swift:77:23: error: Expiring Todo Violation: TODO/FIXME has expired and must be resolved (expiring_todo)
Done linting! Found 1 violation, 1 serious in 640 files.
```

![Image](https://github.com/user-attachments/assets/624a7718-3b65-45be-a2f8-ad905a344fcf)

### Environment

* SwiftLint version: 0.59.1
* Xcode version: Xcode 16.1
* Installation method used: homebrew

**Comment by kaseken** [comment]
I've created a pull request to address this issue.  I’d appreciate it if you could take a look.

https://github.com/realm/SwiftLint/pull/6077
